homeassistant/components/iron_os/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration does not have actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
I am missing tests:
- User step where device already setup
- Bluetooth step where device already setup


Also L51 has 2 periods in the docstring

Reply:
@joostlek https://github.com/home-assistant/core/pull/132265